client/src/locales/en-US.json

```
@@ -410,22 +410,22 @@
         "copySelectFolderTitle": "Copy one item | Copy {count} items",
         "copySelectFolderSubtitle": "Current location: {location}",
         "moveSelectFolderTitle": "Move one item | Move {count} items",
-        "moveSuccess": "The element have been moved the choosen folder. | All the elements have been moved to the chosen folder.",
-        "copySuccess": "The element have been copied to the choosen folder. | All the elements have been copied to the chosen folder.",
+        "moveSuccess": "The element have been moved the chosen folder. | All the elements have been moved to the chosen folder.",
+        "copySuccess": "The element have been copied to the chosen folder. | All the elements have been copied to the chosen folder.",
```

Must use "has" for singular. Also, I would use "selected" instead of "chosen".
```suggestion
        "moveSuccess": "The element has been moved the selected folder. | All the elements have been moved to the selected folder.",
        "copySuccess": "The element has been copied to the selected folder. | All the elements have been copied to the selected folder.",
```

---

Here it's not always a folder that you 'select', maybe 'desired' would be better and cover all use cases we have.